addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/keypad.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dscalarm"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+     <thing-type id="keypad">
+
+	    <supported-bridge-type-refs>
+            <bridge-type-ref id="envisalinkbridge" />
+            <bridge-type-ref id="it100bridge" />
+        </supported-bridge-type-refs>
+
+        <label>DSC Alarm Keypad</label>
+        <description>Represents the central administrative unit of the DSC Alarm System.</description>
+
+        <channels>
+            <channel id="keypad_ready_led" typeId="keypad_ready_led"/>
+            <channel id="keypad_armed_led" typeId="keypad_armed_led"/>
+            <channel id="keypad_memory_led" typeId="keypad_memory_led"/>
+            <channel id="keypad_bypass_led" typeId="keypad_bypass_led"/>
+            <channel id="keypad_trouble_led" typeId="keypad_trouble_led"/>
+            <channel id="keypad_program_led" typeId="keypad_program_led"/>
+            <channel id="keypad_fire_led" typeId="keypad_fire_led"/>
+            <channel id="keypad_backlight_led" typeId="keypad_backlight_led"/>
+            <channel id="keypad_ac_led" typeId="keypad_ac_led"/>
+        </channels>
+
+    </thing-type>
+
+    <channel-type id="keypad_ready_led">
+        <item-type>Number</item-type>
+        <label>Keypad Ready LED</label>
+        <description>Keypad Ready LED (0=Off, 1=On, 2=Flashing)</description>
+        <state min="0" max="2" step="1" readOnly="true"></state>
```

Comment:
Instead of using min, max and step, you should rather directly list the options with `<options>` - you can then even provide labels for the numbers.